rundeckapp/grails-app/services/rundeck/services/NotificationService.groovy

```diff
@@ -787,6 +789,9 @@ public class NotificationService implements ApplicationContextAware{
             execution:[id:exec.id,status:state,user:exec.user],
             notification:[trigger:trigger]
         ]
+        if(export)
+            props << export
```

Comment:
`export` contains a `job` key at this point which ends up replacing the hand-crafted `props` key from line 788. I'm not sure what the impact of that is but:

* It adds a lot more keys to `job`. Was the intention of line 788 to limit the keys available?
* In the case of `group` the null coalesce to empty string gets defeated.